tests/functional/object/rgw/test_obc_quota.py

```
@@ -121,18 +123,467 @@ def test_rgw_obc_quota(
+                pattern="object-",
+                file_dir=test_dir,
+                target=full_bucket_path,
+                amount=upload_amount,
+                s3_obj=obc_obj,
+                ignore_error=False,
+            )
+        except CommandFailed as e:
+            if err_msg in e.args[0]:
+                logger.info("Both quotas hit as expected!!")
+            else:
+                logger.error("ERROR: Copying objects to bucket failed unexpectedly!!")
```

Ruff (0.16.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)